### ISSUE TITLE
[1.0] There is no ripple effect when button is pressed by "Enter" key

### ISSUE THREAD

**Issue by dzhioev** [PolymerElements/paper-icon-button#21]
In contrast to `<paper-button>`, `<paper-icon-button>` doesn't have the ripple effect when pressed by "Enter" key.

**Comment by arthurvr** [comment]
Hmmm... for me there actually seems to be a ripple.

Can you post a reduced testcase? What browser was you using? There are some browser extensions known to be screwing up handlers, so can you try in an incognito window?

**Comment by dzhioev** [comment]
@arthurvr Are you sure, that you used the Enter keyboard key when pressed a button?
Steps to reproduce:
1. Go to https://elements.polymer-project.org/bower_components/paper-icon-button/demo/index.html
2. Focus any button using Tab key.
3. Press Enter or Space key.
My browser is Google Chrome 43.0.2357.81 (Official Build) (64-bit).

**Comment by arthurvr** [comment]
> 2. Focus any button using Tab key.

Indeed. Then I'm able to reproduce this. Try to be a little more complete next time when making a bug report :) If you click once and then do it again by pressing enter there actually is a ripple, strange.

**Comment by dzhioev** [comment]
@notwaldorf Could you please look into this?

**Comment by tjsavage** [comment]
This is also a material design bug - there is no differentiation in the "icon-toggles" section of the spec between `focused` and `pressed` state. 

For non-toggle paper-icon-buttons, I think the solution here should be to replay the ripple animation when keyboard focused and then activated with <Space> or <Enter>, but keep the inky ring visible so that it remains focused.

**Comment by robrez** [comment]
Just to provide some additional info:

 `paper-icon-button-light` has an inverted problem - it _does_ replay the ripple animation when activated,  but removes the inky-focus after the animation has finished.
